nav2_collision_monitor/src/polygon.cpp

```diff
@@ -22,6 +22,7 @@
 
 #include "nav2_util/node_utils.hpp"
 #include "nav2_util/robot_utils.hpp"
+#include "nav2_util/array_parser.hpp"
```

Comment:
Please update the docs: https://navigation.ros.org/configuration/packages/configuring-collision-monitor.html & https://navigation.ros.org/tutorials/docs/using_collision_monitor.html